gatling-http/src/test/scala/io/gatling/http/HttpSpec.scala

```diff
@@ -47,6 +50,7 @@ abstract class HttpSpec extends ActorSpec with BeforeAndAfter with EmptySession
 
   private val clock = new DefaultClock
   protected val mockHttpPort: Int = Using(new ServerSocket(0))(_.getLocalPort).getOrElse(8072)
+  private val nioEventLoopGroup: NioEventLoopGroup = new NioEventLoopGroup()
```

Comment:
Please replace with `private val eventLoopGroup = Transports.newEventLoopGroup(true, false, 1, "test")`.
On Linux (CI), the default channel type is epoll, not NIO, so it's not compatible with NioEventLoopGroup.